testgrid/config.yaml

```diff
@@ -2986,6 +2992,15 @@ dashboards:
     description: Runs conformance tests for cloud provier alibaba cloud on release v1.8
     test_group_name: cloud-provider-aibaba-cloud-e2e-conformance-release-v1.8
 
+# DigitalOcean Dashboard
+- name: conformance-cloud-provider-digitalocean
+  dashboard_tab:
+  - name: DigitalOcean Latest
+    description: Latest DigitalOcean Conformance Results
```

Comment:
cane we describe this a little more inline with the other tests (and add it to the -all dashboard!) I've tried to disambiguate as much as possible what kind of k8s version / build is running under these and on what.

Reply:
done!